internal/helm/chart/builder.go

```diff
@@ -104,6 +104,10 @@ type BuildOptions struct {
 	// Force can be set to force the build of the chart, for example
 	// because the list of ValuesFiles has changed.
 	Force bool
+
+	// Keyring can be configured with the bytes of a public kering in legacy
```

Comment:
```suggestion
	// Keyring can be set to the bytes of a public keyring in legacy
```
Maybe more consistent with the other descriptions above it?